src/content/apis/chatbot/brainshop-ai.md

```diff
@@ -0,0 +1,10 @@
+---
+name: BrainShop.AI
+description: A highly customizable, multilingual chatbot API by Acobot
+categories:
+  - chatbot
```

Comment:
I am not sure if we will many chatbot APIs, so for the time being, can we just change it to other categories.
```suggestion
  - entertainment
  - miscs
```